app/assets/stylesheets/pages/involvement.scss

```diff
@@ -0,0 +1,89 @@
+/* Involvement Styles */
+
+.involvement_page {
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    padding-left: 5cm;
+    padding-right: 5cm;
+    margin-top: 2cm;
```

Comment:
1. Don't use cm, use px! 
2. Instead of doing padding-left, padding-right, just do `padding-left: 0px 5px;` or whatever px is translated from 5cm